exec/cnex/cnex.c

```diff
@@ -75,12 +74,14 @@ BOOL ParseOptions(int argc, char* argv[])
     BOOL Retval = FALSE;
     for (int nIndex = 1; nIndex < argc; nIndex++) {
         if (*argv[nIndex] == '-') {
-            if(argv[nIndex][1] == 'H' || argv[nIndex][1] == 'h' || argv[nIndex][1] == '?' || ((argv[nIndex][1] == '-' && argv[nIndex][2] != '\0') && (argv[nIndex][2] == 'H' || argv[nIndex][2] == 'h'))) {
+            if(argv[nIndex][1] == 'h' || argv[nIndex][1] == '?' || ((argv[nIndex][1] == '-' && argv[nIndex][2] != '\0') && (argv[nIndex][2] == 'h'))) {
```

Comment:
Yes...  It actually does.  It intended to test for -h, -help, --h, --help, all in one shot.  Since I don't have any --hamburger parameters, it just assumed that you meant "help".

Reply:
But what if you did want a `--hamburger` parameter? This line would prevent you from adding one. But adding a `--pizza` parameter would work fine. You should be able to add any new parameters without changing your test for the help command.